docker-compose-LocalExecutor.yml

```diff
@@ -13,8 +13,9 @@ services:
         depends_on:
             - postgres
         environment:
-            - LOAD_EX=y
+            - LOAD_EX=n
             - EXECUTOR=Local
+            - FERNET_KEY=U_HRDNLt20TjStfKkTlopJW7wGcJbExlmknp-m4zzMg=
```

Comment:
its some signing key they use for communication between instances.  This compose file isn't being used. ECS uses its own thing.